docs/cookbook/log-entity-changes.md

```diff
@@ -0,0 +1,136 @@
+!!! danger
+
+    **Now is possible log only entity with primary key `id` and method `getId()`**
+
+```php
+/**
+* @ORM\Table(name="orders")
+* @ORM\Entity
+*/
+#[Loggable(Loggable::STRATEGY_INCLUDE_ALL)]
+class Order
+{
+    ...
+```
+
+or
+
+```php
+/**
+ * @ORM\Table(name="order_items")
+ * @ORM\Entity
+ */
+#[LoggableChild(Loggable::STRATEGY_INCLUDE_ALL)]
+class OrderItem
+{
+    ...
+```
+
+!!! danger
+
+    **Extended entities in the App namespace need to be marked with the Loggable or LoggableChild attribute as well.**
+
+The difference between `Loggable` and `LoggableChild` is in the possibility of assigning a log on the entity marked `LoggableChild` under the logs of another assigned entity. For example, OrderItem is a child entity of the Order entity. In the case of a child entity, it is still necessary to mark its binding property using the `LoggableParentProperty` attribute. In the case of OrderItem, it's the `$order` property.
+
+```php
+/**
+ * @ORM\Table(name="order_items")
+ * @ORM\Entity
+ */
+#[LoggableChild(Loggable::STRATEGY_INCLUDE_ALL)]
+class OrderItem
+{
+    ...
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Order\Order
+     * @ORM\ManyToOne(targetEntity="Shopsys\FrameworkBundle\Model\Order\Order", inversedBy="items")
+     * @ORM\JoinColumn(name="order_id", referencedColumnName="id", nullable=false, onDelete="CASCADE")
+     */
+    #[LoggableParentProperty]
+    protected $order;
+    ...
+}
+```
+
+## Option to name the logged record
+
+The administrator wants to see a human-readable record for the logged record. This is what the `EntityLogIdentify` attribute is used for. This attribute is set on the method that should return the name of the entity. For the OrderItem entity, this is the `getName()` method. Some entities are translatable and you need to mark them like this: `EntityLogIdentify(EntityLogIdentify::IS_LOCALIZED)`. In the background, the administration locale is inserted when such a method is called.
+
+```php
+#[LoggableChild(Loggable::STRATEGY_INCLUDE_ALL)]
+class OrderItem
+{
+    ...
+
+    /**
+     * @return string
+     */
+    #[EntityLogIdentify]
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    ...
+}
+```
+
+There are properties in the Order entity that are not only of scalar data type. For example, the status property is of data type OrderStatus or deliveryCountry is of data type Country (another entity). These entities themselves do not need to be logged in, but we want to see the human name on the order in the event of a status change. The `EntityLogIdentify` attribute is again used for this naming. So if I want to see in the order log that the status has changed from New to In Progress, I need to mark the getName function on the OrderStatus entity.
+
+```php
```

Comment:
the example does not match with the explanation above - the `OrderStatus` is mentioned there, so IMHO the code should show the `OrderStatus` entity, not `Country`

Reply:
thx. fixed